### ISSUE TITLE
main[1] ERROR Expecting value: line 4 column 1 (char 6)

### ISSUE THREAD

**Issue by basq4** [pawliczka/LuxmedSniper#37]
After running docker compose up, application is unable to receive data from portal:

```
luxmedsniper  | 2024-06-27 12:09:21 39fc20eccfc5 LuxMedSniper[1] INFO LuxMedSniper logger initialized
luxmedsniper  | 2024-06-27 12:09:21 39fc20eccfc5 LuxMedSniper[1] INFO Successfully received an access token!
luxmedsniper  | 2024-06-27 12:09:22 39fc20eccfc5 LuxMedSniper[1] INFO Successfully logged in!
luxmedsniper  | 2024-06-27 12:09:22 39fc20eccfc5 main[1] ERROR Expecting value: line 4 column 1 (char 6)
luxmedsniper  | 2024-06-27 12:10:22 39fc20eccfc5 LuxMedSniper[1] INFO LuxMedSniper logger initialized
luxmedsniper  | 2024-06-27 12:10:22 39fc20eccfc5 LuxMedSniper[1] INFO Successfully received an access token!
luxmedsniper  | 2024-06-27 12:10:22 39fc20eccfc5 LuxMedSniper[1] INFO Successfully logged in!
luxmedsniper  | 2024-06-27 12:10:22 39fc20eccfc5 main[1] ERROR Expecting value: line 4 column 1 (char 6)
```

**Comment by marekmaras83** [comment]
+1

**Comment by dungeon-master-office** [comment]
seems like luxmed changed api, requests with token obtained from `PatientPortalMobileAPI` no longer successfull in `PatientPortal/NewPortal` api. i think the only way no is to use `puppeteer` to navigate through patients portal

**Comment by BeniaminK** [comment]
Solved in MR #39
